owl/EASE-STATE.owl

```diff
@@ -27,27 +27,27 @@
     
 
 
-    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#hasGestallt -->
+    <!-- http://www.ease-crc.org/ont/EASE-STATE.owl#hasStateType -->
 
-    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#hasGestallt">
+    <owl:ObjectProperty rdf:about="http://www.ease-crc.org/ont/EASE-STATE.owl#hasStateType">
         <rdfs:subPropertyOf rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#isRelatedToConcept"/>
-        <owl:inverseOf rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#isGestalltOf"/>
+        <owl:inverseOf rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#isStateTypeOf"/>
         <rdfs:domain rdf:resource="http://www.ontologydesignpatterns.org/ont/dul/DUL.owl#Role"/>
-        <rdfs:range rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#Gestallt"/>
+        <rdfs:range rdf:resource="http://www.ease-crc.org/ont/EASE-STATE.owl#StateType"/>
         <rdfs:comment>A relation between roles and gestallts, e.g. &apos;the chair is the supporter of the person sitting on it&apos;.</rdfs:comment>
```

Comment:
Should update comment to say, "A relation between roles and state types"

Reply:
done